xandikos/access.py

```diff
@@ -22,7 +22,6 @@
 See http://www.webdav.org/specs/rfc3744.html
 """
 
-from defusedxml.ElementTree import fromstring as xmlparse
```

Comment:
defusedxml monkeypatches ElementTree, which means it's best to always import defusedxml before importing anything from xml.etree:

morgaine:~% python3  
Python 3.5.3 (default, Jan 19 2017, 14:11:04) 
[GCC 6.3.0 20170118] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from xml.etree import ElementTree as ET1
>>> from defusedxml import ElementTree
>>> from xml.etree import ElementTree as ET2
>>> 
>>> e = ET2.Element('foo')
>>> s = ET1.SubElement(e, 'bar')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: SubElement() argument 1 must be xml.etree.ElementTree.Element, not Element

(this probably warrants a comment for the imports, which I omitted)